features/multiversion.rst

```diff
@@ -3,30 +3,69 @@
 Multiple versions of FHIR
 =========================
 
-Since version 3.0.0 Firely Server can run multiple versions of FHIR side-by-side in the same server. This page explains how this works and what the consequences are.
+Since version 3.0.0, Firely Server can run multiple versions of FHIR side-by-side on the same instance. This page explains how it works and what the consequences are.
+
+Configuration
+-------------
+
+In order to run multiple FHIR versions at the same time, all required binaries need to be loaded upon startup. Include every version in your ``PipelineOptions`` within the appsettings as shown in this example for FHIR STU3, R4 and R5:
```

Comment:
I would highlight that the versions need to be included in the regular pipeline and the admin pipeline.